integration-test-jdbc/src/test/kotlin/integration/jdbc/JdbcDataTypeTest.kt

```diff
@@ -384,6 +390,28 @@ class JdbcDataTypeTest(val db: JdbcDatabase) {
         assertEquals(data, data2)
     }
 
+    @Test
+    fun enum_property() {
+        val m = Meta.enumPropertyData
+        val data = EnumPropertyData(1, Color.BLUE)
+        db.runQuery { QueryDsl.insert(m).single(data) }
+        val data2 = db.runQuery {
+            QueryDsl.from(m).where { m.id eq 1 }.first()
```

Comment:
Shouldn't this be `m.id eq Color.BLUE`?

Reply:
No, this code is correct.
This test case issues following queries:

```sql
insert into enum_property_data (id, "value") values (1, 20)
```
```sql
select t0_.id, t0_."value" from enum_property_data as t0_ where t0_.id = 1
```
You can see that the value property of the Color enum class is mapped to the value column of the enum_property_data table as intended.

Reply:
Maybe the test is right, but would the usage: 
```kotlin
QueryDsl.from(m).where { m.id eq Color.BLUE }.first()
```
also be correct? The test doesn't indicate that... and using `1` has no real meaning in the business logic since the reason why an enum is being used is to abstract that value into the concept of a color... I agree that in certain cases it MIGHT have a meaning such as an RGB value representing a color, but in my case it doesn't and I would not use that value to do my searches.

Reply:
> QueryDsl.from(m).where { m.value eq Color.BLUE }.first()

Don't be worried. This code also works.
The `eq` operator accepts generic types and is tested well.